direnv.el

```diff
@@ -149,13 +181,13 @@ use `default-directory', since there is no file name (or directory)."
 (defun direnv--enable ()
   "Enable direnv mode."
   (--each direnv--hooks
-    (add-hook it #'direnv--maybe-update-environment))
+    (add-hook it 'direnv--maybe-update-environment))
   (direnv--maybe-update-environment))
 
 (defun direnv--disable ()
   "Disable direnv mode."
   (--each direnv--hooks
-    (remove-hook it #'direnv--maybe-update-environment)))
+    (remove-hook it 'direnv--maybe-update-environment)))
```

Comment:
I removed these because (I think?) it makes overriding the hooks from within an editing session just a matter of defining a new function for that symbol. Iiuic using `#'` captures the current value of the function of that symbol, so setting it as a hook means any later updates won't be picked up... is that right??

It would've cost me less time to check than type this comment 😂

Reply:
Nope I just checked and I was wrong, there's no difference. :) It only matters inside `(cl-flet ..)`.